xhr.bs

```diff
@@ -963,6 +967,8 @@ return <a>this</a>'s <a>cross-origin credentials</a>.
  <li><p>If <var>xhr</var>'s <a for=XMLHttpRequest>response</a> is a <a>network error</a>, then
  return.
 
+ <li><p><a for=/>Report timing</a> for <var>xhr</var>.
```

Comment:
Could we move this to line 966, and avoid calling report timing from 2 different places?

Reply:
No, because `handle errors` needs to report errors only in certain conditions + we need to report it if there are no errors. So there will always be two places

Reply:
Sure no probs

Reply:
Ah it wouldn't work because `handle errors` is also called from `processBodyError`, in which case the `processEndOfBody` callback will not be reached at all. We've been through this in early iterations of this PR